data/mods/bio_mod/b_bionics.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "bionic",
+    "id": "bio_emp_armgun",
+    "name": "EMP Projector",
+    "occupied_bodyparts": [
+      [
+        "ARM_R",
+        10
+      ],
+      [
+        "HAND_R",
+        5
+      ]
+    ],
+    "act_cost": 20,
+    "gun_bionic": true,
+    "fake_item": "bio_emp_gun",
+    "description": "A ranged emp generator system is implanted on the palm of the user's right hand and arm.  Fires super-concentrated electric pulses for a short distance.  Extremely effective against electronic targets but mostly useless otherwise."
+  },
+  {
+    "type": "bionic",
+    "id": "bio_chest_gun",
+    "name": "Ionic Overload Generator",
+    "occupied_bodyparts": [
+      [
+        "TORSO",
+        20
+      ]
+    ],
+    "act_cost": 500,
+    "gun_bionic": true,
+    "fake_item": "bio_ion_gun",
+    "description": "A powerful ion energy generator is implanted on the user's chest.  Fires a powerful, ever-expanding energy blast.  The resulting blast ignites oxygen creating fires as it moves and an explosion on impact. Close range use is highly discouraged."
+  },
+  {
+    "id": "bio_surgical_razor",
+    "type": "bionic",
+    "name": "Autonomous Surgical Razors",
+    "description": "Implanted on the user's fingers is a system of surgical grade razors.  While activated, they will continously drain power to make automated precise cuts but you will be unable to wield anything.",
+    "occupied_bodyparts": [
+      [ "ARM_R", 1 ],
+      [ "ARM_L", 1 ]
+    ],
+    "toggled": true,
+    "act_cost": 10,
+    "react_cost": 1,
+    "time": 1,

```

Comment:
The reason being is that it's meant do the cutting for you. Also, it's not mean to be a bionic you would have active very much, only when you need butchering since as a weapon it's not that great.